doc/locale_api_WIP.md

```diff
@@ -0,0 +1,48 @@
+
+Note: ListFoo APIs combine the IDs and the translations. 
+If it's easier to return just the IDs and ask for the translations separately,
+let's use such different API.
+
+### Locales
+
+```
+ListLocales(
+  in  s     display_locale # "en_US.UTF-8"
+  out a(ss) id_label_pairs
+)
+
+SetLocale(in s locale_id)
+```
+
+### Keyboards
+
+```
+ListX11Keyboards(
+  in  s     display_locale # "en_US.UTF-8"
+  out a(ss) id_label_pairs  # id like "layout(variant)"
+)
+
+SetX11Keyboard(in s kb_id)
+
+X11KeyboardForLocale(
+  in  s locale_id
+  out s kb_id
+)
+```
+
+### Timezones
+
+
+```
+ListTimezones(
+  in  s     display_locale # "en_US.UTF-8"
+  out a(ss) id_label_pairs
+)
+
+SetTimezone(in s tz_id)
+
+TimezoneForLocale(
+  in  s locale_id
+  out s tz_id
+)
+```
```

Comment:
There will be probably one `Commit` method to write all of them down. I also miss here some method to get locale/keyboard/timezone as UI should display what user already selected.